src/kbmod/search/KBMOSearch.cpp

```diff
@@ -10,14 +10,13 @@
 namespace search {
 
 #ifdef HAVE_CUDA
-    extern "C" void deviceSearchFilter(int imageCount, int width, int height, float* psiVect, float* phiVect,
-                                       perImageData img_data, searchParameters params, int trajCount,
-                                       trajectory* trajectoriesToSearch, int resultsCount,
-                                       trajectory* bestTrajects);
-
-    void deviceGetCoadds(ImageStack& stack, perImageData image_data, int num_trajectories,
-                         trajectory* trajectories, stampParameters params,
-                         std::vector<std::vector<bool> >& use_index_vect, float* results);
+extern "C" void deviceSearchFilter(int num_images, int width, int height, float* psi_vect, float* phi_vect,
+                                   PerImageData img_data, SearchParameters params, int num_trajectories,
+                                   trajectory* trj_to_search, int num_results, trajectory* best_results);
+
```

Comment:
Unrelated to the current PR, just a general comment. These big blocks of params make it very difficult for me to see the changes. I know it's not C++ style but if we could do a one-per-line after 5 params that would make it really easy to see what changes in a PR and what the func expects. As well as allow inline annotations if/when we need them.